core/src/main/java/org/mule/runtime/core/internal/policy/CompositeOperationPolicy.java

```diff
@@ -140,4 +150,11 @@ public CoreEvent process(CoreEvent event) throws MuleException {
       return error(e);
     }
   }
+
+  private void logOperationEvent(String startingMessage, EventContext eventContext, Message message) {
+    if (LOGGER.isTraceEnabled()) {
```

Comment:
The policy name is not being logged anywhere for operation policies. Why don't you put the logs in the same place where it is in the CompositeSourcePolicy and with the same format?

Reply:
eventContext is the only object that has the information that we need to log, and the only method that expose them is toString(). It prints it in a slightly different format than we do, but as the method was already defined I didn't dare to modify it

Reply:
But why log where you are logging and not where I am saying? It is not useful to log that if you don't know in which policy you are